src/main/routes/watch.ts

```diff
@@ -30,7 +30,7 @@ export default function (app: Application): void {
 
       if (recording === null) {
         res.status(404);
-        res.render('not-found');
+        res.render('not-available');
```

Comment:
From reading the jira ticket I think we need to distinguish between a recording which just doesn't exist (404 not-found) and a recording which did exist but belongs to a now closed case (404 not available).
I think this change will display (404 not available) for both of these use cases. You'd need to see if the API returns a different message for these use cases in it's response (I'm not sure off the top of my head). There may need to be a change in the API response to be able to distinguish between the 2 scenarios.

Reply:
Thanks, have updated as we discussed and have updated the ticket. I've removed the functional tests now as it relies on having a closed case, not sure on your approach for testing data being held in STG but if its left forever then I could add it back.